src/Microsoft.Health.Fhir.Core/Features/Persistence/RawResource.cs

```diff
@@ -31,5 +32,8 @@ protected RawResource()
         [JsonProperty("format")]
         [JsonConverter(typeof(StringEnumConverter))]
         public FhirResourceFormat Format { get; protected set; }
+
+        [JsonProperty("metaSet")]
+        public bool MetaSet { get; protected set; }
```

Comment:
Minor naming suggestion: `IsMetaSet` (along with some explanatory comments).

Reply:
Sure. Should I do this throughout (so in the sql column name too)?

Reply:
I made the changes in the sql column names too, made sense to have it match between data stores and match the convention of prefixing with is for bit fields.